.github/workflows/documentation.yml

```diff
@@ -1,12 +1,10 @@
 name: Documentation
-on:
-  push:
-    branches:
-      - master
+on: [push]
 
 jobs:
   build-and-publish-docfx:
     runs-on: ubuntu-latest
+    if: github.repository == 'freezy/VisualPinball.Engine' && github.ref == 'refs/heads/master' 

```

Comment:
Why compare to the repo? To avoid forks deploying?

Reply:
correct. My `gh_pat` had permissions and it looked like it deployed successfully when I was working out of my repo.